src/test/java/io/reactivex/internal/operators/flowable/FlowableLastTest.java

```diff
@@ -54,35 +53,35 @@ public void testLastViaObservable() {
 
     @Test
     public void testLast() {
-        Flowable<Integer> observable = Flowable.just(1, 2, 3).last();
+        Single<Integer> observable = Flowable.just(1, 2, 3).last();
 
-        Subscriber<Integer> observer = TestHelper.mockSubscriber();
+        SingleObserver<Integer> observer = TestHelper.mockSingleObserver();
         observable.subscribe(observer);
 
         InOrder inOrder = inOrder(observer);
-        inOrder.verify(observer, times(1)).onNext(3);
-        inOrder.verify(observer, times(1)).onComplete();
+        inOrder.verify(observer, times(1)).onSuccess(3);
+//        inOrder.verify(observer, times(1)).onComplete();
```

Comment:
Will come in handy when the fuse-back is implemented (by copying these and restoring them to Observer/Subscriber). I'll delete them then